docs/user-guide/reading-and-writing-files.ipynb

```diff
@@ -0,0 +1,83 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reading and Writing Files\n",
+    "\n",
+    "## HDF5\n",
+    "\n",
+    "Scipp support writing variables, data arrays, and dataset to HDF5 files.\n",
+    "Reading of HDF5 is supported *only* for these scipp-specific files.\n",
+    "Other HDF5-based formats are not supported at this point.\n",
+    "For reading NeXus files, see [scipp-neutron](../scipp-neutron/overview.html).\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "We do not recommend to use scipp HDF5 files for archiving or as the sole means of storing valuable data.\n",
+    "The current scipp HDF5 file format is not a standard and will likely be subject to change due to the early development status of scipp.\n",
+    "**Future versions of scipp may not be able to read older files.**\n",
+    "    \n",
+    "That being said, the file format is quite simple and based on the HDF5 standard so it would still be possible to recover data from such files in such a case.\n",
```

Comment:
Might we mention that scipp writes the scipp version used to write the file into the output?